js/blogtini.js

```diff
@@ -279,12 +290,28 @@ async function fetcher(url)  {
 async function main() {
   let tmp
 
-  // see if this is an (atypical) "off site" page/post, compared to the main site
-  // eslint-disable-next-line no-use-before-define
-  const [my_frontmatter] = markdown_parse(document.getElementsByTagName('body')[0].innerHTML)
-  const base = my_frontmatter?.base
+  /**
+   * Pick all content of the page, let's figure out what to do with it.
+   * Beware, if we use innerHTML and want the front-matter not to be with escaped entities
+   * we got to use textContent instead.
+   */
+  const textContent = document.getElementsByTagName('body')[0].textContent
+  const [raw_fm] = splitFrontMatterAndMarkdown(textContent)
+
+  let my_frontmatter = {}
+  try {
+    const loaded = yml.load(raw_fm)
+    my_frontmatter = loaded
+  } catch (e) {
+    console.error('blogtini main ERROR loading YAML', e)
+    my_frontmatter = {}
+  }
+
+  const base = my_frontmatter?.base ?? SITE_ROOT_BASE_URL
+  const href = window.location.href
+  const maybe = href.replace(base ?? '', '')
 
-  state.pathrel = state.is_homepage ? '' : '../' // xxxx generalize
+  state.pathrel = state.is_homepage ? '' : maybe // xxxx generalize
```

Comment:
Things around here are hard to figure out, I would suggest to simplify (what I was trying to understand with `maybe`) that or organize them differently.